src/main/java/io/debezium/connector/vitess/VitessConnectorConfig.java

```diff
@@ -222,22 +224,23 @@ public static BigIntUnsignedHandlingMode parse(String value, String defaultValue
             .withWidth(Width.MEDIUM)
             .withImportance(ConfigDef.Importance.HIGH)
             .withDescription(
-                    "Single shard of which keyspace to read data from."
-                            + "E.p. \"0\" for an unsharded keyspace. "
-                            + "Or \"-80\" for the -80 shard of the sharded keyspace.");
+                    "Shard(s) of which keyspace to read data from."
+                            + "E.g. \"0\" for an unsharded keyspace. "
+                            + "Or \"-80\" for the -80 shard of the sharded keyspace."
+                            + "Or \"-4000,4000-8000\" for two of the four shards of a sharded keyspace.");
 
-    public static final Field GTID = Field.create(VITESS_CONFIG_GROUP_PREFIX + "gtid")
-            .withDisplayName("gtid")
+    public static final Field VGTID = Field.create(VITESS_CONFIG_GROUP_PREFIX + "vgtid")
```

Comment:
Will it cause any regression with the config name changes from `gtid` to `vgtid`?

Reply:
Yes, it will. It is necessary to deprecate the existing setting. So both should be available and when the deprecated one is used then WARN should be written to the log.